.yarn/versions/9f1d25ab.yml

```diff
@@ -0,0 +1,31 @@
+releases:
+  "@yarnpkg/builder": patch
```

Comment:
You only need a patch release for
- core
- link
- http
- git
- file
- exec
- cli